source/opt/inline_pass.h

```diff
@@ -117,11 +134,31 @@ class InlinePass : public Pass {
                      ir::UptrVectorIterator<ir::Instruction> call_inst_itr,
                      ir::UptrVectorIterator<ir::BasicBlock> call_block_itr);
 
-  // Returns true if |inst| is a function call that can be inlined.
+  // Return true if |inst| is a function call that can be inlined.
   bool IsInlinableFunctionCall(const ir::Instruction* inst);
 
-  // Returns true if |func| is a function that can be inlined.
-  bool IsInlinableFunction(const ir::Function* func);
+  // Compute structured successors for function |func|.
+  // A block's structured successors are preceded by its
```

Comment:
Yes. This algorithm is a bit clunky. I probably could mention that extra merge blocks in the successor list are harmless.